core/store/migrations/migration1607954593/migrate.go

```diff
@@ -0,0 +1,18 @@
+package migration1607954593
+
+import "github.com/jinzhu/gorm"
+
+func Migrate(tx *gorm.DB) error {
```

Comment:
looks worth of a quick migration unit test?

Reply:
What for?

Reply:
we don't actually test this update statement anywhere

Reply:
(not a blocking comment)